app/build.gradle

```diff
@@ -36,7 +36,11 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.7.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+    //location service library
+    implementation'com.google.android.gns:play-services-location:17.0.0'
 
+    //Notification library
+    implementation "com.android.support:support-compat:28.0.0"
```

Comment:
I added Notification library to your build.gradle because your applications board indicated creating a notification alert when the user's location is close to a new activity they have selected as an interest.
 Check this out for more details: https://developer.android.com/training/notify-user/build-notification#kts